avidemux_fr.ts

```diff
@@ -9943,12 +9943,12 @@ L&apos;édition en cours a été enregistrée et sera rechargée au prochain dé
     <message>
         <location line="+122"/>
         <source>Load video filters..</source>
-        <translation>Charger les filtres vidéo..</translation>
+        <translation>Charger les filtres vidéos..</translation>
```

Comment:
1. I wonder whether the original translation was more correct (i.e. no agreement between "filtre" and "vidéo" in plural necessary), seen on numerous web pages like https://shotcut.fr/
2. More important, this translation has no effect as the corresponding code is excluded from compilation by preprocessor directives. lupdate still collects such translatable strings (this can be by design, unsure about it).

Reply:
It is true that the habit is to consider video as invariable as an adjective.
Since the 1990 reform, in the expression "video filters", the term "videos" has been considered as an adjective; it therefore agrees in number with the substantive. 

Reforms of French orthography, Wikipedia : https://en.wikipedia.org/wiki/Reforms_of_French_orthography#Rectifications_of_1990